.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
to ignore

Reply:
it is for VScode eslint plugin. It needs to be told if the current workspace is using new eslint config style